01_derive_data.R

```diff
@@ -459,7 +488,8 @@ p_clst_char <- lapply(colnames(dat_clust)[!colnames(dat_clust) %in% c("lake",
     p <- p + scale_y_log10()
   }
   return(p)
-  }) |> ggarrange(plotlist = _)
+  })
+ggarrange(plotlist = p_clst_char)
```

Comment:
I think you need to change this to `p_clst_char <- ggarrange(plotlist = p_clst_char)` otherwise the wrong plot will be saved in line 495